IDEAS/Buildings/Components/Interfaces/RectangularZoneTemplateInterface.mo

```diff
@@ -1304,6 +1346,11 @@ components cannot be propagated.
 </html>", revisions="<html>
 <ul>
 <li>
+August 28, 2018, by Damien Picard:<br/>
+Changes to allow multiple external surfaces connection per faces.
+See <a href=\"https://github.com/open-ideas/IDEAS/issues/901\">#901</a>.
+</li>
+<li>
 August 26, 2018, by Damien Picard:<br/>
 Move all equations except windows equations of
 RectangularZoneTemplate to this interface model.
```

Comment:
@damienpicard we discussed asserts that verify that the number of propsbusses is 1 when using an internal wall. Can you add these?

Reply:
Done